src/test/java/com/palantir/docker/compose/DockerComposeRuleShould.java

```diff
@@ -224,9 +231,14 @@ public void be_able_to_save_logs_to_a_directory_while_containers_are_running()
 
     @Test
     public void not_shut_down_when_skipShutdown_is_true() throws InterruptedException {
-        defaultBuilder().skipShutdown(true)
-                        .build()
-                        .after();
+        DockerComposeRule.builder()
```

Comment:
`skipShutdown` only takes a boolean, but we are setting `shutdownStrategy` a bit unnecessarily.  Have scoped it down to the one test method that needs it.